new/LIT7106Shinbone.xml

```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7106Shinbone" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ አቁያጻት፡</title>
```

Comment:
Is it better to give the "correct" አቍያጻት, ʾaqʷyāṣāt, in the title? I guess that is what people would be searching for if they are looking for parallels in other works. Or is it better to keep the spelling of the manuscript? I'm not sure :)

Reply:
Yes, it is better to spell it according to Leslau in the titles.